cli/iconset-builder.js

```diff
@@ -17,7 +17,7 @@ var iconsetObjTemplate = {
 };
 
 function processFill(elem) {
-	if (elem.$ && elem.$.fill && elem.$.fill !== 'none') {
+	if (elem.$ && elem.$.fill && elem.$.fill !== 'none' && elem.$.fill.toLowerCase() === '#72777a') {
```

Comment:
You can likely remove the check for `none` now that it checking specifically for that color.